documentation/book/api/paths.adoc

```diff
@@ -936,6 +936,32 @@ Check if the bridge is ready and can accept requests.
 |===
 
 
```

Comment:
This is great! 
What do you think about the ordering of the methods in the OpenAPI spec? It seems odd to document the /Topics endpoints after the GET methods for `healthy`, `openapi`, and `ready`. I think the following order is more logical:
```
# ...

GET /topics * NEW

GET /topics/{topicname} * NEW

POST /topics/{topicname}

POST /topics/{topicname}/partitions/{partitionid}

GET /healthy

GET /openapi

GET /ready
```

Reply:
I think we have no control over the ordering because the spec is currently written in json and the ordering of its entries is not preserved in general. I suppose, one way to preserve the ordering is to write the spec in yaml.

Reply:
OK, thanks for looking into it. It's not a big deal, so let's leave the order unchanged.